apps/meteor/ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -215,6 +214,88 @@ export const LivechatEnterprise = {
 		await AutoCloseOnHoldScheduler.unscheduleRoom(roomId);
 		await LivechatRooms.unsetOnHoldAndPredictedVisitorAbandonmentByRoomId(roomId);
 	},
+
+	/**
+	 * @param {string|null} _id - The department id
+	 * @param {Partial<import('@rocket.chat/core-typings').ILivechatDepartment>} departmentData
+	 * @param {{upsert?: { agentId: string; count?: number; order?: number; }[], remove?: { agentId: string; count?: number; order?: number; }[]}} [departmentAgents] - The department agents
+	 */
+
```

Comment:
JSDoc comments *MUST NOT* be apart from the commented statement, even by a empty line.
```suggestion
```